spring-cloud-skipper-client/src/main/java/org/springframework/cloud/skipper/client/DefaultSkipperClient.java

```diff
@@ -117,10 +117,18 @@ public AboutInfo info() {
 	}
 
 	@Override
-	public Info status(String releaseName) {
+	public Info status(String releaseName, Integer releaseVersion) {
```

Comment:
Should we add an overloaded method instead?  That is provide
`status(String releaseName)` as well as `status(String releaseName, Integer releaseVersion)` in which case we would assert not null for releaseVersion?